fhir/practitioner/routes/practitioner.ts

```diff
@@ -0,0 +1,40 @@
+import * as express from 'express';
+import * as codes from '../../patient/controller/errorCodes';
+import { Auth } from '../../../auth/auth.class';
+import * as utils from '../../../utils/utils';
+import { profesional } from '../../../core/tm/schemas/profesional';
+import { encode } from '@andes/fhir/src/practitioner';
```

Comment:
Estaría bueno organizar los exports del modulo para que funcione así:

```suggestion
import { Practitioner } from '@andes/fhir';

...

Practitioner.encode(dto);
Practitioner.decode(dto);
```

Reply:
Dale, me parece bien. Ahora lo modifico